packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 
+import 'package:devtools_app/src/ui/flutter/service_extension_widgets.dart';
```

Comment:
make this a relative path. There is a quickfix for that.

Reply:
Done.